landing-page/main.css

```diff
@@ -0,0 +1,65 @@
+        :root {
+            --blue: #0a81d1;
+            --black: #1b1b1d;
+        }
+        
+        
+        .main_wrapper {
+            background: var(--black);
+            width: 90vw;
+            height: 90vh;
+            display: grid;
+            grid-template-columns: repeat(5, 2fr);
+            grid-template-rows: repeat(5, 1fr);
+            font-family: 'Roboto' , sans-serif;
+            color: white;
+        }
+        
+        .prompt {
+            grid-column: 6 / span 3;
+            grid-row: 2 / span 3;
+        }
+
+        .heading {
+            font-size: 26px;
+        }
+        
+        .heading, p {
+            font-size: 22px;
+        }
+        
+        .input {
+            height: 46px;
+            width: 412px;
+            border-radius: 5px;
+            border: 0px;
+        }
+
+        .input:focus {
+            border: 2px solid var(--blue);
+        }
+        
+        .button {
+            height: 46px;
+            width: 128px;
+            border-radius: 5px;
+            border: none;
+        }
+
+        .signup_link, a{
+            margin: 15px 0px;
+            text-decoration: none;
+            color: var(--blue);
+        }
+ 
+
+        .button {
+            color: white;
+            margin-top: 55px;
+            font-weight: 500;
+            background: var(--blue);
+            font-size: 13px;
+        }
+        
+        a:hover, .button:hover {
+            opacity: 0.75;
```

Comment:
Make sure to close your brackets when creating css classes. Just add a `}` at the end of this file.